src/revlanguage/functions/math/Func_absVectorInt.h

```diff
@@ -0,0 +1,48 @@
+#ifndef Func_absVectorInt_H
+#define Func_absVectorInt_H
+
+#include "ModelVector.h"
+#include "Natural.h"
+#include "RlTypedFunction.h"
+
+#include <string>
+
+namespace RevLanguage {
+    
+    /**
+     * The RevLanguage wrapper of the absolute value function.
+     *
+     * The RevLanguage wrapper of the abs function connects
+     * the variables/parameters of the function and creates the internal AbsoluteValueFunction object.
+     * Please read the AbsoluteValueFunction.h for more info.
+     *
+     *
+     * @copyright Copyright 2009-
+     * @author The RevBayes Development Core Team (Sebastian Hoehna)
+     * @since 2014-07-27, version 1.0
```

Comment:
Technically, this is inaccurate insofar as it concerns this particular file rather than RevBayes as a whole... Again, not a big deal, but I can imagine situations where knowing who added a particular function might be useful.

Reply:
Fixed by removing that block.